### ISSUE TITLE
Not working with yarn

### ISSUE THREAD

**Issue by steebchen** [h2non/requireg#8]
When I use yarn and do `sudo yarn global add ip`, I can't require a global module (module not found). If I do `npm i -g ip` however, it works. Is there a simple way that requireg detects global packages installed by yarn? I am also not sure whether this is an issue with requireg but I can use any global installed package (with yarn) per command line.

**Comment by mselerin** [comment]
A simple resolver for yarn : 

```javascript
function yarnModulesResolve(module) {
    let i, modulePath;

    // Retrieve yarn global path
    let yarnCmd = isWin32 ? 'yarn.cmd' : 'yarn';
    let result = spawnSync(yarnCmd, ['global', 'dir'], { encoding: 'utf8' });
    let yarnPath = result.stdout;
    yarnPath = yarnPath.replace(/[\r\n]+/g, '');

    let nodeModulesStr = path.join(yarnPath, 'node_modules');

    if (typeof nodeModulesStr === 'string') {
        let nodeModules = nodeModulesStr.split(path.delimiter);
        for (i = 0; i < nodeModules.length; i++) {
            if (modulePath = resolveFn(module, nodeModules[i])) {
                break;
            }
        }
    }

    return modulePath;
}
```

Works fine :-)

**Comment by h2non** [comment]
Can you provide a PR?

**Comment by mselerin** [comment]
Of course, as soon as I can !

**Comment by h2non** [comment]
This has been fixed in `v0.2.0`. Closing.
